app/models/runtime/helpers/space_role_mixin.rb

```diff
@@ -0,0 +1,22 @@
+module VCAP::CloudController
+  module SpaceRoleMixin
+    def self.included(included_class)
+      included_class.class_eval <<-RUBY, __FILE__, __LINE__ + 1
```

Comment:
Is there a reason to use this syntax rather than a do...end block?

Reply:
We had to do this to for the special rails associations methods.
`many_to_one` don't translate the same as normal functions because they are rails magic functions that will eventually inject association methods.

Unless you are saying there is another special thing to inject these rail methods into our mixin?

Reply:
We could do it this way but I have no idea what extra stuff we get by adding ActiveSupport::Concern will give us.
```
module VCAP::CloudController
    module SpaceRoleMixin
        extend ActiveSupport::Concern

        included do
          many_to_one :user
          many_to_one :space

          def_column_alias :guid, :role_guid
        end
...
```

Reply:
We discussed this in the sync, but adding here for posterity:

`class_eval` can either take a string or block.  So this could be rewritten like so: 

```
included_class.class_eval do
     many_to_one :user
     many_to_one :organization
     def_column_alias :guid, :role_guid
end
```

However, I am not sure if you can still pass in the file/line number for the backtrace with the block method and how that behavior will be different.